Orange/widgets/data/owimpute.py

```diff
@@ -202,8 +202,8 @@ def __init__(self):
         box = gui.auto_commit(
             self.controlArea, self, "autocommit", "Apply",
             orientation=Qt.Horizontal, checkbox_label="Apply automatically")
-        box.layout().insertSpacing(0, 80)
-        box.layout().insertWidget(0, self.report_button)
+        box.button.setFixedWidth(180)
```

Comment:
Why is here this line and not also in **Impute** widget (https://github.com/biolab/orange3/pull/2669/files#diff-a2cbec3457e61cdb3228e0e4b7b95585R233)?

Reply:
You mean Discretize? Because I didn't like any of them and I couldn't decide which I disliked less. If a widget is wide, there is no good position for the Apply button.

Now I how changed (unified) the layout of Discretize, Color, Concatenate and Impute to look the same and reasonably nice.